tests/pc/test_po_update.py

```diff
@@ -58,6 +58,14 @@ def reload_testbed_on_failed(request, duthosts, enum_rand_one_per_hwsku_frontend
         config_reload(duthost, safe_reload=True, ignore_loganalyzer=loganalyzer)
 
 
+def bgp_enabled(duthost):
+    return 'BGP instance not found' not in duthost.shell('show ip bgp neighbors')['stdout']
```

Comment:
What if the BGP service is down? In this case, "show ip bgp neighbors" will also output something like `BGP instance not found`. Could this lead to false positive test result? Probably the better way is to check if bgp service is enabled or not using command `systemctl`.

Reply:
The original PR title, content and code do not state the case clearly. BGP feature is enabled while LAG connects to peer directly without engaging BGP.

I refined PR again saying there is no BGP neighbors.